### PR TITLE
Add starting point for {arp,cwd,srs} dictionaries

### PR DESCRIPTION
**Pull request by andrewdotn**
There are definitely issues with this code but at least this will let other
people run the demo with multiple dictionary languages on their machines.

Good:

  - Three new language pairs!
  - Docs for how to load dictionary data
  - There’s a `newdictsite` command to create all the files for a new
    language

Not so good:

  - The `urls.py` is copied everywhere
  - Lots of other copy-paste
  - `newdictsite` is likely to easily get out of date, but is better than
    nothing
  - Dictionaries are not yet intelligent

### CODE REVIEW

File: src/srseng/site/settings.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+MORPHODICT_SOURCE_LANGUAGE = "srs"
+MORPHODICT_TARGET_LANGUAGE = "eng"
+
+MORPHODICT_ORTHOGRAPHY = {
+    "default": "Latn",
+    "available": {
+        "Latn": {"name": "Latin"},
+    },
+}
```
Comment (eddieantonio) [inline src/srseng/site/settings.py:47]: I wonder if it's okay to omit the `MORPHODICT_ORTHOGRAPHY` if a language has only one orthography.

File: src/cwdeng/dictimport/management/commands/cwdimport.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    def handle(self, *args, json_file, **options):
+        with open(json_file, "r") as f:
+            dictionary_data = json.load(f)
+
+        root = ET.Element("r")
+
+        source = ET.Element("source")
+        source.set("id", "LLR")
+        title = ET.Element("title")
+        title.text = "Lac La Longe"
```
Comment (eddieantonio) [inline src/cwdeng/dictimport/management/commands/cwdimport.py:40]:
```suggestion
        title.text = "Lac La Ronge"
```

**Review by eddieantonio — APPROVED** [review]
One typo I noticed, plus a few minor concerns, but otherwise, this is good to merge!

File: src/srseng/site/settings.py (hunk 1 above)
Comment (andrewdotn) [reply src/srseng/site/settings.py:47]: At the moment, it crashes :/

File: src/cwdeng/site/settings.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        "Latn": {"name": "SRO (êîôâ)"},
+        "Latn-x-macron": {
+            "name": "SRO (ēīōā)",
+            "converter": "CreeDictionary.CreeDictionary.orthography.to_macrons",
+        },
+        "Cans": {
+            "name": "Syllabics",
+            "converter": "CreeDictionary.CreeDictionary.orthography.to_syllabics",
+        },
+    },
```
[the remark this answers is not in the document]
Comment (andrewdotn) [reply src/cwdeng/site/settings.py:63]: I did not know that about the macrons, thank you! CMRO will come later.

File: src/arpeng/site/urls.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+from django.contrib.sitemaps.views import sitemap
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+from django.urls import include, path
+from django_js_reverse.views import urls_js
+
+import CreeDictionary.API.views as api_views
+from CreeDictionary.CreeDictionary import views
+from CreeDictionary.CreeDictionary.sitemaps import sitemaps
+
+# TODO: use URL namespaces:
```
[the remark this answers is not in the document]
Comment (andrewdotn) [reply src/arpeng/site/urls.py:16]: This is totally fixable in the future. But for now, you are totally right, making four exact copies of the same single file that badly needs refactoring is not doing any good.

File: src/cwdeng/site/settings.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (dwhieb) [reply src/cwdeng/site/settings.py]: Did not know this either! 😲

File: src/crkeng/site/settings.py [hunk 5]
```diff
@@ -25,15 +25,16 @@
...
 
 # Morphodict configuration
 
 # The ISO 639-1 code is used in the lang="" attributes in HTML.
 MORPHODICT_ISO_639_1_CODE = "cr"
 
+MORPHODICT_SOURCE_LANGUAGE = "crk"
+MORPHODICT_TARGET_LANGUAGE = "eng"
+
 # What orthographies -- writing systems -- are available
```
Comment (dwhieb) [inline src/crkeng/site/settings.py:38]: Pedantic point, totally just an FYI: Writing systems and orthographies are slightly different. A writing system is a group of associated symbols, like the Latin alphabet. An orthography is how a particular writing system is _used_ to write a particular language. So the Spanish and English _orthographies_ both use the Latin _writing system_.

The ISO codes are for writing systems, and the subcodes allow you to specify variants of the writing system. For example, the variant of the Latin writing system that has acute accents on vowels is used for the Spanish orthography, while the variant of the Latin writing system without accents on vowels is used for the English orthography.

File: src/cwdeng/site/settings.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (aarppe) [reply src/cwdeng/site/settings.py]: Nevertheless, note that Woods folks such as Sol Ratt have written `cwd` texts with circumflexes, cf.

https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/827

Also, according to Arok, SRO was open to using a variety of diacritics for marking long vowels, which would not have been restricted to circumflexes and macrons but could include accents (aigu), if those are easier to type, i.e. `á`, `é`, `í`, or `ó`.

**Pull request merged**